soda/core/soda/execution/profile_columns_run.py

```diff
@@ -141,41 +144,63 @@ def profile_numeric_column(
             )
             aggregates_query.execute()
             if aggregates_query.rows is not None:
-                profile_columns_result_column.average = float(aggregates_query.rows[0][0])
-                profile_columns_result_column.sum = float(aggregates_query.rows[0][1])
-                profile_columns_result_column.variance = float(aggregates_query.rows[0][2])
-                profile_columns_result_column.standard_deviation = float(aggregates_query.rows[0][3])
-                profile_columns_result_column.distinct_values = int(aggregates_query.rows[0][4])
-                profile_columns_result_column.missing_values = int(aggregates_query.rows[0][5])
+                profile_columns_result_column.average = (
+                    float(aggregates_query.rows[0][0]) if aggregates_query.rows[0][0] is not None else None
```

Comment:
can't we create a helper method like;

```
def cast_float_with_none(float_number):
     if float_number is not None:
          return float(float_number)
     return None

```

Reply:
good thinking @baturayo I'll take care of that indeed.